eqasc/code/README.md

```diff
@@ -0,0 +1,124 @@
+## eQASC Evaluator
+
+This script evaluates predictions for eQASC predictions against ground truth annotations and produces metrics.
+
+Hint: If you are in a hurry and want to simply evaluate your predictions, run the evaluator in Docker.
+
+## Usage
+
+The program [evaluator.py](allennlp_reasoning_explainqa/evaluator/evaluator.py) takes three arguments:
+
+1. The filename of a prediction file.
+2. The filename of the labels to evaluate against.
+3. The filename where metrics will be written.
+
+### Prediction file
+
+The predictions file should hold multiple JSON objects, with each object having a score, and a chain ID. For example:
+
+```bash
+%  cat predictions/grc.test.predict | head -n 4
+{"score": 0.2023383378982544, "chain_id": "3C44YUNSI1OBFBB8D36GODNOZN9DPA_1_1"}
+{"score": 0.5158032774925232, "chain_id": "3C44YUNSI1OBFBB8D36GODNOZN9DPA_1_2"}
+{"score": 0.17925743758678436, "chain_id": "3C44YUNSI1OBFBB8D36GODNOZN9DPA_1_5"}
+{"score": 0.8793290853500366, "chain_id": "3C44YUNSI1OBFBB8D36GODNOZN9DPA_1_7"}
+```
+
+The chain IDs must match those in the labels. (See below.)
+
+The file `predictions/grc.test.predict` in this repo contains an example
+prediction for the test labels. It was made by doing ... TODO
```

Comment:
@harsh19 can you explain how this file was made?

Reply:
Using command at https://github.com/harsh19/Reasoning-Chains-MultihopQA/blob/evaluator/code/scripts/grc.sh#L43

Reply:
Thanks! I'll update the TODO.